stackit/internal/services/ske/kubeconfig/resource.go

```diff
@@ -116,6 +119,13 @@ func (r *kubeconfigResource) Schema(_ context.Context, _ resource.SchemaRequest,
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"kubeconfig_uuid": schema.StringAttribute{
```

Comment:
Maybe name it `kubeconfig_id` for consistency with other resources (e.g. `instance_id`)